### PR TITLE
DBZ-1621 IoUtil Bugfix	

### PR DESCRIPTION
**Pull request by pan3793**
https://issues.jboss.org/browse/DBZ-1621

### CODE REVIEW

File: debezium-core/src/main/java/io/debezium/util/IoUtil.java [hunk 1]
```diff
@@ -250,22 +250,20 @@ public static InputStream getResourceAsStream(String resourcePath,
             resourceDesc = resourcePath;
         }
         InputStream result = null;
-        if (result == null) {
```
Comment (gunnarmorling) [inline debezium-core/src/main/java/io/debezium/util/IoUtil.java]: Weird. Good catch :)

**Comment by gunnarmorling** [comment]
Good catch, thanks, @pan3793! Could you only rework the commits with an e-mail that's [associated to your GH account](https://help.github.com/en/github/setting-up-and-managing-your-github-profile/why-are-my-contributions-not-showing-up-on-my-profile)? Thanks!

**Comment by pan3793** [comment]
Thanks for your detailed review, e-mail was fixed.

**Comment by gunnarmorling** [comment]
Applied. Thx, @pan3793!

**Pull request merged**